bc_obps/registration/api/user_operator.py

```diff
@@ -42,6 +43,14 @@ def is_approved_admin_user_operator(request, user_guid: str):
     return 200, {"approved": approved_user_operator}
 
 
+@router.get("/user-operator-id/{user_guid}", response={200: UserOperatorIdOut, codes_4xx: Message})
```

Comment:
I think we could probably use the current_user middleware here instead of a param since we just need the guid to look up the operator, not for any encrypted reasons